lib/hello.js

```diff
@@ -5,7 +5,11 @@ var cors = require('cors');
 function helloRoute() {
   var hello = new express.Router();
   hello.use(cors());
-  hello.use(bodyParser());
+  //hello.use(bodyParser());
```

Comment:
remove the comment.
This change is for removing the deprecate warnings? In other templates we didn't update body-parser so we didn't encounter the deprecate warnings and I was going to create a new issue about updating templates for using latest dependencies